bridge/bridge.go

```diff
@@ -282,3 +324,9 @@ func (b *Bridge) didExitCleanly(containerId string) bool {
 	}
 	return !container.State.Running && container.State.ExitCode == 0
 }
+
+var Hostname string
+
+func init() {
+	Hostname, _ = os.Hostname()
```

Comment:
Might we worth a comment that it's OK for hostname to be an empty string, because we're handling it above. If there were more logic based on Hostname, this could get dangerous